ServerCore/Pages/Index.cshtml

```diff
@@ -15,7 +15,7 @@
                     Intern Puzzleday 2021<br />
                 </div>
                 <div style="text-align:center;padding-top:6px;">
-                    <img src="~/images/divider.png" />
+                    <img src="~/images/divider.png" />
```

Comment:
I can't tell for sure what changed here, but I didn't intentionally touch this line.

Reply:
VS complained about line endings in this file when I opened it prior to my last deploy. I am guessing it's related to that, and that you actually cleaned it up.